x-pack/plugins/security_solution/public/overview/components/overview_empty/index.test.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import { OverviewEmpty } from '.';
+import { useIngestEnabledCheck } from '../../../common/hooks/endpoint/ingest_enabled';
+jest.mock('../../../common/lib/kibana');
+jest.mock('../../../management/pages/endpoint_hosts/view/hooks', () => ({
+  useIngestUrl: jest
+    .fn()
+    .mockReturnValue({ appId: 'ingestAppId', appPath: 'ingestPath', url: 'ingestUrl' }),
+}));
+
+jest.mock('../../../common/hooks/endpoint/ingest_enabled', () => ({
+  useIngestEnabledCheck: jest.fn().mockReturnValue({ allEnabled: true }),
+}));
+
+jest.mock('../../../common/hooks/endpoint/use_navigate_to_app_event_handler', () => ({
+  useNavigateToAppEventHandler: jest.fn(),
+}));
+
+describe('OverviewEmpty', () => {
+  describe('When isIngestEnabled = true', () => {
+    let wrapper: ShallowWrapper;
+    beforeAll(() => {
+      wrapper = shallow(<OverviewEmpty />);
+    });
+
+    afterAll(() => {
+      (useIngestEnabledCheck as jest.Mock).mockReset();
+    });
+
+    test('render with correct actions ', () => {
+      expect(wrapper.find('[data-test-subj="empty-page"]').prop('actions')).toEqual({
+        beats: {
+          description:
+            'Leightweight Beats can send data from hundreds or thousands of machines and systems',
+          fill: false,
+          label: 'Add data with Beats',
+          url: '/app/home#/tutorial_directory/security',
+        },
+        elasticAgent: {
+          description:
+            'The Elastic Agent provides a simple, unified way to add monitoring to your hosts.',
+          fill: false,
+          label: 'Add data with Elastic Agent',
+          url: 'ingestUrl',
+        },
+        endpoint: {
+          description:
+            'Protect your hosts with threat prevention, detection, and deep security data visibility.',
+          fill: false,
+          label: 'Add Elastic Endpoint Security',
+          onClick: undefined,
+          url: 'ingestUrl',
```

Comment:
With regards to the behavior of the `Add Elastic Endpoint Security` button pictured in the screenshot below:

![welcome](https://user-images.githubusercontent.com/4459398/88598653-f5d60500-d026-11ea-956a-072d5ab8a255.png)


Perhaps for consistency with the `Add data with Elastic Agent` button, the `Add Elastic Endpoint Security` button links to the more general `Integrations` page pictured in the screenshot below:

![integrations](https://user-images.githubusercontent.com/4459398/88598236-1782bc80-d026-11ea-8cbe-7eae7a2422e6.png)

Thus, when users click `Add Elastic Endpoint Security`, it still requires an additional click on the `Elastic Endpoint Security` button, pictured above, to finally arrive at the actual `Elastic Endpoint Security` page, pictured in the screenshot below:

![elastic-endpoint-security](https://user-images.githubusercontent.com/4459398/88598451-89f39c80-d026-11ea-860e-499a2d3c8042.png)

As more integrations are added, the `Elastic Endpoint Security` button (on the `Integrations` page) may be harder to find. It also may not be obvious to users that an addition click is required (on the `Integrations` page).

Should the `Add Elastic Endpoint Security` button link directly to the `Elastic Endpoint Security` page?

Reply:
@andrew-goldstein I like the idea of the more direct path to the actual Elastic Endpoint Security integration page.  The only downside is that a user may not get the context that this is one of many integrations available, but given that they user comes from the "Add  Elastic Endpoint Security" button, I think this is a good tradeoff.

I'd like to defer to @bradenlpreston since he's been studying this onboarding workflow and can provide clearer guidance.

Reply:
@caitlinbetz for confirmation:

My opinion - we should link directly to the Endpoint Security integration page.

Reply:
@bradenlpreston @MikePaquette The `Elastic Endpoint Security` page displays an error message if a specific version is omitted from the URL, as shown in the screenshot below:

<img width="1694" alt="error" src="https://user-images.githubusercontent.com/4459398/88728937-fd151580-d0ef-11ea-8337-e68f988e8d07.png">

 Thus we can't directly link to the `Elastic Endpoint Security` page (in this PR) without creating a brittle touch point (with the potential to link to a stale/old version).

### Details

On `master`, the `Elastic Endpoint Security` button in the `Ingest Manager` `integrations` catalog, shown in the screenshot below...

<img width="1709" alt="integrations" src="https://user-images.githubusercontent.com/4459398/88730088-888fa600-d0f2-11ea-8d0d-a3bea6785789.png">


...has a URL that links to version `0.11.0` of the `Elastic Endpoint Security`. Specifically, the URL is:

```
http://localhost:5601/app/ingestManager#/integrations/detail/endpoint-0.11.0
``` 

These version numbers may increment frequently, (it would have linked to `0.9.0` just a few days ago), and they are likely to change at least once every release.

To ensure the link always goes to the current version, the link should be non-version-specific.

As noted above, if we omit the version in the URL, changing it from

```
http://localhost:5601/app/ingestManager#/integrations/detail/endpoint-0.11.0
```

to

```
http://localhost:5601/app/ingestManager#/integrations/detail/endpoint
```

The non-version-specific `Elastic Endpoint Security` page _does_ exist (it's not a `404`), but it displays the following error (in the screenshot below) when the version is omitted from the URL:

<img width="1694" alt="error" src="https://user-images.githubusercontent.com/4459398/88728937-fd151580-d0ef-11ea-8337-e68f988e8d07.png">

The screenshot above illustrates that the page served by the non-version-specific URL "knows about" the current version (0.11.0), but it displays an error.

To ensure this PR is included in the next BC (and doesn't introduce a brittle touch point), we will merge it with the original behavior.